clients/src/main/java/org/apache/kafka/common/requests/AlterShareGroupOffsetsRequest.java

```diff
@@ -65,7 +65,8 @@ public AbstractResponse getErrorResponse(int throttleTimeMs, Throwable e) {
                 .setPartitions(topicResult.partitions().stream()
                     .map(partitionData -> new AlterShareGroupOffsetsResponseData.AlterShareGroupOffsetsResponsePartition()
                         .setPartitionIndex(partitionData.partitionIndex())
-                        .setErrorCode(Errors.forException(e).code()))
+                        .setErrorCode(Errors.forException(e).code())
```

Comment:
`Errors.forException()` scans the list of Errors matching on exception class. I would prefer this to be done just once, such as assigning a local Errors variable and then using it to set the error code and message on each look iteration.